From 2e00916e0f925da2d109c218ce457a1695794974 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Fri, 31 May 2024 10:24:51 +0200
Subject: [PATCH 0889/1695] dts: st: add scmi-agent-id in scmi_regu node in
 stm32mp211.dtsi

After patch "core: scmi-server: regulator: use agent_id and channel_id
from parent", nodes with compatible "st,scmi-regulator-consumer" should at
least have a scmi-agent-id property and a scmi-channel-id.

Update stm32mp211.dtsi

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 6d59a9cb4d61dd36960a0feea9affbbb4bf6dcf1 dts: st: add scmi-agent-id in scmi_regu node in stm32mp211.dtsi]
Co-developed-by: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Signed-off-by: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I2ab2d7e2be79e9839fba8e0bd272676c837274bb
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/384366
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 16f4b6e56..39cb92f0c 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -133,6 +133,7 @@
 		compatible = "st,scmi-regulator-consumer";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		scmi-agent-id = <0>;
 		scmi-channel-id = <0>;
 
 		voltd-vddio1 {
-- 
2.47.2

